web/packages/teleport/src/services/storageService/storageService.ts

```diff
@@ -102,6 +102,21 @@ export const storageService = {
     return time ? time : 0;
   },
 
+  setLoginTime(time: number) {
```

Comment:
Do we need to pass an argument? I think `setLocalTime` can always read `new Date.now()`.